yarn.lock

```diff
@@ -6574,7 +6569,7 @@ min-indent@^1.0.0:
   resolved "https://registry.yarnpkg.com/min-indent/-/min-indent-1.0.1.tgz#a63f681673b30571fbe8bc25686ae746eefa9869"
   integrity sha512-I9jwMn07Sy/IwOj3zVkVik2JTvgpaykDZEigL6Rx6N9LbMywwUSMtxET+7lVoDLLd3O3IXwJwvuuns8UB/HeAg==
 
-minimatch@^3.0.2, minimatch@^3.0.4:
+minimatch@^3.0.4:
```

Comment:
Im curious why yarn.lock changed here 😅 
Besides that looks great

Reply:
Sorry! I backed out the changes to yarn.lock!

Reply:
@diegomura thats weird. I backed out these changes and simply reinstalled the packages and this still comes through. How would you like to resolve this?